submissions/franchukv/a-tiny-js-world/index.js

```diff
@@ -0,0 +1,77 @@
+/* 
+   Refer to https://github.com/OleksiyRudenko/a-tiny-JS-world for the task details
+   Complete the below for code reviewers' convenience:
+   Code repository: https://github.com/franchukv/a-tiny-JS-world
+   Web app: https://franchukv.github.io/a-tiny-JS-world/
+*/
+
+// ======== OBJECTS DEFINITIONS ========
+const man = {
+   species: 'human',
+   gender: 'male',
+   name: 'Drogo',
+   status: 'Khal',
+   hands: 2,
+   feets: 2,
+   saying: 'Sheikh Ma Shieraki Anni.',
+}
+
+const woman = {
+   species: 'human',
+   gender: 'female',
+   name: 'Daenerys',
+   status: 'Daenerys of House Targaryen, First with Her Name, Breaker of Chains and Mother of Dragons',
+   hands: 2,
+   feets: 2,
+   saying: 'Drakaris!',
+   friends: `Fell in love with ${man.name}.`,
+}
+
+const dog = {
+   species: 'dog',
+   gender: 'male',
+   name: 'Sharik',
+   status: "Yard's terrier",
+   hands: 0,
+   feets: 4,
+   saying: 'woof-woof!',
+   friends: `Only ${man.name} is friends.`,
+}
+
+const cat = {
+   species: 'cat',
+   gender: 'female',
+   name: 'Bastet',
+   status: "Pharaoh's cat",
+   hands: 0,
+   feets: 4,
+   saying: 'Meow, bow to me!',
+   friends: 'No friends, only servants!'
+}
+
+const catwoman = {
+   species: 'human',
+   gender: 'female',
+   name: 'Selina Kyle',
+   status: "Superhero",
+   hands: 2,
+   feets: 2,
+   saying: cat.saying,
+   friends: `In astral connection with ${cat.name}.`,
+}
+
+// ======== OUTPUT ========
+const inhabitants = [man, woman, dog, cat, catwoman];
+const properties = ['species', 'gender', 'name', 'status', 'hands', 'feets', 'saying', 'friends'];
+
+function objectToString(habitant) {
+   let temporaryArray = [];
+
+   properties.forEach((property) => {
+      temporaryArray.push(habitant[property]);
+   })
+
+   return temporaryArray.join('; ');
```

Comment:
rows 68-74 are classic use case for `Array#map`. Learn it. You will often use it.